scoap3/articles/api/views.py

```diff
@@ -34,22 +35,7 @@ class ArticleViewSet(
 ):
     queryset = Article.objects.all()
     serializer_class = ArticleSerializer
-    permission_classes = [IsAuthenticatedOrReadOnly]
-
-    def create(self, request, *args, **kwargs):
```

Comment:
we have updated the `created` method to check if article already exists and return error.. we should keep this

Reply:
But the we also should the check for update and delete as well, am I right?

Reply:
Like this:
https://github.com/SCOAP3/scoap3/blob/7fb8196a77645e07179c903b0072779259a9fba1/scoap3/articles/api/views.py#L59-L74